Source/WebCore/Modules/indexeddb/server/MemoryIDBBackingStore.h

```diff
@@ -46,6 +46,7 @@ class MemoryIDBBackingStore final : public IDBBackingStore {
     IDBError getOrEstablishDatabaseInfo(IDBDatabaseInfo&) final;
     uint64_t databaseVersion() final;
     void setDatabaseInfo(const IDBDatabaseInfo&);
+    bool hasObjectStore(uint64_t objectStoreIdentifier) { return !!infoForObjectStore(objectStoreIdentifier); }
```

Comment:
Can this be marked as const?

Reply:
Nope, because `infoForObjectStore` is not (it looks like `infoForObjectStore` should be const though, just need to change all functions it calls to be const and confirm...).